.bazelci/presubmit.yml

```diff
@@ -328,9 +328,15 @@ tasks:
     platform: ubuntu2004
     working_directory: examples
     build_targets:
+      - "--" # Allows negative patterns; hack for https://github.com/bazelbuild/continuous-integration/pull/245
       - "//..."
+      # Nix example is only supported on NixOS container
+      - "-//nix_cross_compiling/..."
```

Comment:
@UebelAndre is there a better way to exclude the Nix example from all the non-NixOS platforms?

Reply:
Hmm looks like this doesn't even work as it still tries to load the `WORKSPACE.bazel`.

Reply:
Update the working directory and make sure you add the nix directory to the examples .bazelignore file